src/node.cc

```diff
@@ -1923,6 +1923,63 @@ static void Uptime(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+static void GetResourceUsage(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  uv_rusage_t rusage;
+  int err = uv_getrusage(&rusage);
+  if (err)
+    return env->ThrowUVException(err, "uv_getrusage");
+
+  Local<Object> resource_usage = Object::New(env->isolate());
```

Comment:
@bnoordhuis @trevnorris 
What if someone was to query resource stats on a regular basis (like 1/500ms), i.e. to build a graph? In that case the overhead of passing the args back the _normal_ way would have considerable perf impact right? 
And the only upside to doing it the less performant way is slightly less ugly code.